packages/roosterjs-editor-plugins/lib/TabIndent/TabIndent.ts

```diff
@@ -83,6 +87,33 @@ export default class TabIndent implements EditorPlugin {
                     }
                 }
             }
+        } else {
+            let nodeAtCursor = getNodeAtCursor(this.editor);
+            let [isBlockQuoteEvent, blockQuoteElement] = this.isBlockQuoteEvent(event, [KEY_ENTER], nodeAtCursor);
+            if (isBlockQuoteEvent) {
+                let keybordEvent = (event as PluginDomEvent).rawEvent as KeyboardEvent;
+                keybordEvent.preventDefault();
+                let childCount = blockQuoteElement.childNodes.length;
+                this.editor.deleteNode(nodeAtCursor);
+                let range = this.editor.getSelectionRange();
+                if (!!range) {
+                    range.setEndAfter(blockQuoteElement);
+                    range.collapse(false);
+                }
+                let brNode = this.editor.getDocument().createElement('br');
+                this.editor.updateSelection(range);
+                this.editor.insertNode(brNode, {
+                    position: ContentPosition.SelectionStart,
+                    updateCursor: false,
+                    replaceSelection: true,
+                    insertOnNewLine: false,
+                });
+
+                // If the current node is the only child in blockquote, delete the blockquote
```

Comment:
Could you provide more examples about this case?

Reply:
<blockquote><div><br></div></blockquote>
If user enter on <br>, we delete the <br> node and move cursor out of the blockquote, in such situation we need to delete the blockquote node as well since it has not content now.